### ISSUE TITLE
Unable to point aria-controls to the autocomplete, as there is no ID

### ISSUE THREAD

**Issue by nichols-green** [swiftype/swiftype-wordpress#64]
In order to meet required accessibility standards, we need to add a tag of `aria-controls="<ID-of-autocomplete-container>"` to the input field of our search. The container does not have an ID and cannot be referenced.
